src/link/SpirV.zig

```diff
@@ -30,16 +35,15 @@ const trace = @import("../tracy.zig").trace;
 const build_options = @import("build_options");
 const spec = @import("../codegen/spirv/spec.zig");
 
+// TODO: Should this struct be used at all rather than just a hashmap of aux data for every decl?
```

Comment:
Noted. I have a plan to clean up `FnData` in the near future, feel free to just leave this messy and I'll get to it.